kotlin/kotlin.bzl

```diff
@@ -38,7 +38,6 @@ load(
     _kt_js_library = "kt_js_library_macro",
 )
 
-define_kt_toolchain = _define_kt_toolchain
```

Comment:
Can you validate that this is what you mean? For instance, we use this to define alternate pulls of kotlinc (so we don't have to upgrade the rules to get a minor version bump) and is in the docs. If we're moving it, it's a thing we should document in the docs/relnotes.